spice_ev/strategies/schedule.py

```diff
@@ -195,7 +196,7 @@ def evaluate_core_standing_time_ahead(self):
         self.charge_window = [x > 0 for x in self.power_for_vehicles_per_TS]
         TS_to_charge_vehicles = sum(self.charge_window)
 
-        # PV and Grid energy available for vehicles
+        # Local Generation and Grid energy available for vehicles
```

Comment:
```suggestion
        # local generation and grid energy available for vehicles
```

Reply:
Now: "power from local generation and grid power available for vehicles"